.github/workflows/specs.yml

```diff
@@ -18,6 +18,14 @@ jobs:
       - name: tests/fast/*
         run: source ~/.rvm/scripts/rvm && tf --text tests/fast/*
       - name: named_ruby_and_gemsets_comment_test
+        # works on local, but fails in CI, needs to be investigated
+        if: ${{ !contains(matrix.os, 'macos') }}
         run: source ~/.rvm/scripts/rvm && tf --text tests/long/named_ruby_and_gemsets_comment_test.sh
+      - name: ruby_prepare_mount_comment_test
+        # https://github.com/rvm/rvm/issues/4937
+        if: ${{ !contains(matrix.os, 'macos') }}
+        run: source ~/.rvm/scripts/rvm && tf --text tests/long/ruby_prepare_mount_comment_test.sh
       - name: truffleruby_comment_test
+        # works on local, but fails in CI, needs to be investigated
```

Comment:
Do you know what failed in CI?
I would expect this test to pass on macOS.
I'll try removing the `if` and see what happens.